genesis/cmd/main.go

```diff
@@ -43,42 +42,28 @@ func worldgenMain(ctx *cli.Context) error {
 		logger.Info("monorepo artifacts download progress", "current", curr, "total", total)
 	}
 
-	monorepoOPCMConfig := &bootstrap.OPCMConfig{
-		ArtifactsLocator: &artifacts.Locator{
-			URL: cliConfig.MonorepoArtifactsURL,
-		},
+	monorepoArtifactsLocator := &artifacts.Locator{
+		URL: cliConfig.MonorepoArtifactsURL,
 	}
 
-	monorepoArtifactsFS, monorepoArtifactsCleanup, err := artifacts.Download(ctx.Context, monorepoOPCMConfig.ArtifactsLocator, monorepoProgressor)
+	monorepoArtifactsFS, err := artifacts.Download(ctx.Context, monorepoArtifactsLocator, monorepoProgressor)
 	if err != nil {
 		return fmt.Errorf("failed to download monorepo artifacts: %w", err)
 	}
-	defer func() {
```

Comment:
`artifacts.Download` no longer returns a cleanup func, so we no longer need this: https://github.com/ethereum-optimism/optimism/commit/c38ebfc011a6f9dff86316df7ee2e3da4e7b352e#diff-4cded8e4d1164e5eb23ca79aa00a0c4cd1b8ed429c46c5b4e40bd6c647cd1a01L163